sdks/java/io/file-based-io-tests/build.gradle

```diff
@@ -31,8 +31,7 @@ dependencies {
   testCompile project(path: ":sdks:java:io:parquet", configuration: "testRuntime")
   testCompile project(path: ":sdks:java:testing:test-utils", configuration: "testRuntime")
   testCompile library.java.jaxb_api
-  testCompile library.java.jaxb_impl
   testCompile library.java.junit
-  testCompile library.java.hamcrest_core
   testCompile library.java.hadoop_client
+  permitUnusedDeclared "org.checkerframework:checker-qual:3.7.0"
```

Comment:
Choosing one file at random to comment on: do this in BeamModulePlugin not in every build.gradle file.

Reply:
I had no idea about this, it works like magic! :D a lot of repeated stuff got removed. Thanks for this recommendation!